gtsam/inference/EliminateableFactorGraph-inst.h

```diff
@@ -36,7 +36,7 @@ namespace gtsam {
       // no Ordering is provided.  When removing optional from VariableIndex, create VariableIndex
       // before creating ordering.
       VariableIndex computedVariableIndex(asDerived());
-      return eliminateSequential(orderingType, function, computedVariableIndex);
+      return eliminateSequential(orderingType, function, &computedVariableIndex);
```

Comment:
should be restored after using reference wrapper

Reply:
done